indexclient/client.py

```diff
@@ -600,6 +600,30 @@ def version(self) -> str:
     def version(self, new_version: str) -> None:
         self._doc["version"] = new_version
 
+    @property
+    def uploader(self) -> str:
```

Comment:
nit pick:
Wouldn't this fail if mypy was on?
`dict.get()` returns `None` when key is absent.